app/views/catalog/_show_coin_description.html.erb

```diff
@@ -0,0 +1,2 @@
+<% doc_presenter = show_presenter(document) %>
+<%= render CoinDescriptionComponent.new fields: doc_presenter.field_presenters.select { |field| field.field_config&.coin_description }, show: true %>
```

Comment:
It would be nice for the ruby CoinDescriptionComponent class to do this filtering, vs putting it in the view.